src/App.js

```diff
@@ -15,9 +26,17 @@ function App() {
 
   return (
     <div className="container">
-      <Logo />
+      <Logo className="logo" />
+      <div className="displayContainer">
+        <Display result={number} />
+      </div>
       <div className="App">
         {/* STEP 4 - Render your components here and be sure to properly import/export all files */}
+        <div className="buttonContainer">
+          <Specials number={number} setNumber={setNumber} />
+          <Numbers number={number} setNumber={setNumber} />
```

Comment:
Instead of passing down the `setNumber` function as a prop it would be better if we passed down a function that contained the logic that this component is going to use. Something like `(num) => setNumber(number => number+num)`. That way, we are keeping those components dumb. They don't contain any more logic than they need to. They don't care about the implementation details of the `setNumber` function.